packages/core/src/config-manager/index.ts

```diff
@@ -323,4 +302,60 @@ export default class ConfigManager implements IConfigManager {
         adminRoutes.getModulesRoute(this.serviceDiscovery.registeredModules),
       );
   }
+
+  private registerConfigRoutes(moduleName: string, configSchema: any) {
```

Comment:
Can we at at least use `convict.Config<any>` here?
Won't fix CodeFactor, but it should be clearer for anybody reading the code.